README.md

```diff
@@ -0,0 +1,86 @@
+# Sandbox Engineering sb03
+
+The goal of this class is to prepare you to quickly build and scale products and systems.
+
+## 3 core parts of a product
+
+You can break down most software products into 3 parts:
+
+ - `Frontend` (the software on the user's device)
+ - `Backend` (the systems that you control)
+ - `Datastore` (anything that stores data)
+
+## The Stack
+
+### Language: TypeScript
+Starting with the language, we will be using TypeScript. TypeScript is a superset of JavaScript. JavaScript is the only language that natively runs in the browser and that has led to its massive popularity. Technologies such as Node.js have allowed JavaScript to also run on the server as well. This allows to use one language across the frontend and backend.
+
+### Frontend: React
+React is a JavaScript framework for building user interfaces. It basically uses JavaScript to generate HTML on the client's device (i.e. you have a variable that is your user's name and React can insert that JavaScript variable into the HTML to display it on the screen).
+
+```TypeScript
+function MyButton() {
+  const [count, setCount] = useState(0);
+
+  function handleClick() {
+    setCount(count + 1);
+  }
+
+  return (
+    <button onClick={handleClick}>
+      Clicked {count} times
+    </button>
+  );
+}
+```
+
+### Backend: Next.js
+Next.js is like a fusion of React and a backend framework in one. It allows you to run React on the server and then send the rendered HTML to the client. It also allows you to create API endpoints that can be called from the frontend.
+
+```TypeScript
+import { NextResponse } from 'next/server'
+ 
+export async function GET() {
+  const res = await fetch('https://data.myapi.com/...', {
```

Comment:
Can we do a fake prisma call here instead? There's an example in my Lecture 3 notes.